ci-operator/step-registry/rosa/cluster/provision/rosa-cluster-provision-commands.sh

```diff
@@ -579,32 +583,45 @@ CLUSTER_ID=$(cat "${CLUSTER_INFO}" | grep '^ID:' | tr -d '[:space:]' | cut -d ':
 echo "Cluster ${CLUSTER_NAME} is being created with cluster-id: ${CLUSTER_ID}"
 echo -n "${CLUSTER_ID}" > "${CLUSTER_ID_FILE}"
 
-# Watch the hypershift install log
-if [[ "$HOSTED_CP" == "true" ]]; then
-  rosa logs install -c ${CLUSTER_ID} --watch
-fi
```

Comment:
we cannot watch the installation process, because if we watch for it, command will hold until cluster will be installed to end, and we lose the possibility of getting the timers. 

after the loop, there is a rosa logs install, which is valid for both rosa classic and hosted clusters, so the functionality is the same than setting a watch

Reply:
@yasun1 can you confirm deleting this watch is ok?